tests/test_examples.py

```diff
@@ -218,8 +218,8 @@ def run_qh_step(n, eq):
 
     objective = ObjectiveFunction(
         (
-            QuasisymmetryTwoTerm(helicity=(1, -eq.NFP), grid=grid, normalize=False),
-            AspectRatio(target=8, weight=1e1, normalize=False),
+            QuasisymmetryTwoTerm(helicity=(1, eq.NFP), grid=grid),
```

Comment:
I think the helicity should be negative for this example: `helicity=(1, -eq.NFP)`.

Reply:
Since we changed the sign of the Jacobian, it changed the sign of theta for this case. So the helicity needs to change as well. Technically it should be (-1, -NFP) but figured this was less confusing.

Reply:
OK yes I checked it and that's correct that the helicity is positive now.